src/components/02_atoms/Widgets/BooleanCheckbox.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CheckBox from '@material-ui/core/Checkbox';
+import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
+
+const BooleanCheckbox = props => (
+  <CheckBox
+    id={props.fieldName}
+    checked={props.checked}
+    onChange={event => props.onChange(event.target.value)}
+    margin="normal"
+    label={props.label}
```

Comment:
label is not a valid property here 

can I suggest wrapping checkbox in FormControlLabel
I am also thinking about how to make this extendable and want to pass any extra properties like color="primary" down into the checkbox element.

I hope this helps

  render() {
    /* Pass all props not used at this level down into Checkbox */
    const { onChange, label, ...checkboxProps } = this.props;
    return (
      <FormControlLabel
        control={
          <Checkbox
            {...checkboxProps}
            checked={this.state.checked}
            onChange={onChange}
            value=""
          />
        }
        label={label}
      />
    );
  }

Reply:
Great remark @martinfrances107 !